pkg/plugins/testfreeze/testfreeze.go

```diff
@@ -35,10 +35,21 @@ const (
 	PluginName                  = "testfreeze"
 	defaultKubernetesBranch     = "master"
 	defaultKubernetesRepoAndOrg = "kubernetes"
-	templateString              = `Please note that we're already in [Test Freeze](https://github.com/kubernetes/sig-release/blob/master/releases/release_phases.md#test-freeze) for the ` + "`{{ .Branch }}`" + ` branch. This means every merged PR will be automatically fast-forwarded via the periodic [ci-fast-forward](https://testgrid.k8s.io/sig-release-releng-blocking#git-repo-kubernetes-fast-forward) job to the release branch of the upcoming {{ .Tag }} release.
+	templateString              = `{{ if .InCodeFreeze }}Please note that we're already in [Code Freeze](https://github.com/kubernetes/sig-release/blob/master/releases/release_phases.md#code-freeze) for the upcoming {{ .Tag }} release.
+
+**Adding the milestone to this PR is strictly prohibited without proper approval.** If this PR needs to be included in the {{ .Tag }} release:
+1. The corresponding SIG must approve the PR (` + "`/lgtm`" + ` or ` + "`/approve`" + `)
+2. Get approval from the Release Team Lead
+3. Then a milestone maintainer can add the ` + "`{{ .Branch }}`" + ` milestone
+
+/cc @kubernetes/release-team-leads @kubernetes/release-managers
+{{ if .InTestFreeze }}
+---
+{{ end }}{{ end }}{{ if .InTestFreeze }}
+We're also in [Test Freeze](https://github.com/kubernetes/sig-release/blob/master/releases/release_phases.md#test-freeze) for the ` + "`{{ .Branch }}`" + ` branch. This means every merged PR will be automatically fast-forwarded via the periodic [ci-fast-forward](https://testgrid.k8s.io/sig-release-releng-blocking#git-repo-kubernetes-fast-forward) job to the release branch of the upcoming {{ .Tag }} release.
```

Comment:
Per discussion in https://github.com/kubernetes-sigs/prow/issues/557

cc @pohly @kubernetes-sigs/release-engineering